src/DetailsView/components/reflow-assessment-view.tsx

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import * as React from 'react';
+import { ContentLinkDeps } from 'views/content/content-link';
+
+import { AssessmentTestResult } from '../../common/assessment/assessment-test-result';
+import { Tab } from '../../common/itab';
+import {
+    AssessmentData,
+    AssessmentNavState,
+    PersistedTabInfo,
+} from '../../common/types/store-data/assessment-result-data';
+import { GettingStartedView } from './getting-started-view';
+import { TargetChangeDialog, TargetChangeDialogDeps } from './target-change-dialog';
+
+export type WithAssessmentTestResult = { assessmentTestResult: AssessmentTestResult };
+
+export type ReflowAssessmentViewDeps = ContentLinkDeps & TargetChangeDialogDeps;
```

Comment:
Is the contentlinkdeps being used?

Reply:
No it's not, so going to remove it! Moving forward, should I continue using `ReflowAssessmentViewDeps`? 

so option 1:  `export type ReflowAssessmentViewDeps = TargetChangeDialogDeps;'  and keep props as is
option 2: delete `ReflowAssessmentViewDeps` and updated the props to say `deps: TargetChangeDialogDeps`